src/Benchmarks/Program.cs

```diff
@@ -44,11 +46,16 @@ public static void Main(string[] args)
                     .AddSingleton<Scenarios>()
                 );
 
-            Server = "kestrel";
-
             var webHost = webHostBuilder.Build();
-
-            StartInteractiveConsoleThread();
+            
+            Console.WriteLine($"Using server {Server}");
+            Console.WriteLine($"Server GC is currently {(GCSettings.IsServerGC ? "ENABLED" : "DISABLED")}");
+            
+            var nonInteractiveValue = config[NonInteractiveKey];
```

Comment:
When running automated in the techempower framework it needs to be a background task, so it doesn't actually have a console.  In that case console.readkey throws and server doesn't start.